styles/Hero.module.css

```diff
@@ -0,0 +1,68 @@
+.container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding-bottom: var(--spacing-unit);
+}
+
+.hero_container {
+    margin-top: 319.5px;
```

Comment:
Please use CSS vars

Reply:
No variable in the CSS globals that reference this box position